web/src/components/auth/NeedsAccountModal.tsx

```diff
@@ -43,48 +45,51 @@ export const NeedsAccountModal = (): ReactElement => {
 
   return (
     <Dialog
+      fullWidth={false}
       open={showNeedsAccountModal}
-      maxWidth="xs"
+      maxWidth="sm"
       onClose={(): void => setShowNeedsAccountModal(false)}
       data-testid={"self-reg-modal"}
+      aria-labelledby="modal"
+      disableEnforceFocus={contextualInfo.isVisible}
     >
-      <DialogTitle sx={{ p: 5, paddingRight: 10 }}>
-        <Heading level={0} styleVariant="h3">
+      <DialogTitle id="modal" className="display-flex flex-row flex-align-center margin-top-1 break-word">
+        <Heading level={0} styleVariant="h2" className="padding-x-1 margin-0-override">
```

Comment:
[dust] This renders it as a `<div>` rather than a heading element. Are we sure we want to do that here?

Reply:
discussed, no changes will be made